drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata-app.yaml

```diff
@@ -119,3 +119,30 @@ spec:
       resources:
         requests:
           storage: 5Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: es-load
+  namespace: default
+  labels:
+    app: es-load
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: es-load
+  template:
+    metadata:
+      labels:
+        app: es-load
+    spec:
+      containers:
+      - name: es-load
+        image: serhiiqbox/px-enterprise:es-stress-test
```

Comment:
change this to not use your personal image, ask @shailvipx to create a repository like portworx/es-sidecar

Reply:
That was used for testing purposes. Updated to `portworx/torpedo-esload` but I can't push the image (have no access permissions).